### PR TITLE
Use shorthand functions for DOM operations more often

### PR DESCRIPTION
**Pull request by Fabii547**

### CODE REVIEW

File: wcfsetup/install/files/js/WoltLab/WCF/Date/Picker.js [hunk 1]
```diff
@@ -686,14 +686,14 @@ define(['DateUtil', 'Language', 'ObjectMap', 'Dom/ChangeListener', 'Ui/Alignment
...
-			container.parentNode.removeChild(container);
+			elRemove(container);
 			
 			elAttr(element, 'type', 'date' + (data.isDateTime ? 'time' : ''));
 			element.value = data.shadow.value;
 			
 			element.removeAttribute('data-value');
 			element.removeEventListener('click', _callbackOpen);
-			data.shadow.parentNode.removeChild(data.shadow);
+			elRemove(data);
```
Comment (Gravatronics) [inline wcfsetup/install/files/js/WoltLab/WCF/Date/Picker.js]: This looks wrong.

Comment (Fabii547) [inline wcfsetup/install/files/js/WoltLab/WCF/Date/Picker.js]: Fixed with 4e1d7912205aae47aaba1c7ba5669c83979739f2

**Comment by dtdesign** [comment]
@Fabii547 Would you be so kindly to rebase your pull request?

**Comment by Fabii547** [comment]
Done.

**Comment by Fabii547** [comment]
@dtdesign Should now work properly.

**Comment by dtdesign** [comment]
@Fabii547 Perfect, thank you for your help!

**Pull request merged**